packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -665,6 +665,13 @@ export abstract class FluidDataStoreContext
 		}
 	}
 
+	public addedGCOutboundRoute(fromPath: string, toPath: string) {
```

Comment:
Nit: `addedGCOutboundReference` so everything matches

Reply:
I named it differently on purpose - it has a different signature.  I'll make sure there's comments everywhere to highlight that.